mobile/android/chrome/browser.properties

```diff
@@ -278,12 +292,28 @@ masterPassword.incorrect=Nieprawidłowe hasło
 # LOCALIZATION NOTE (remoteIncomingPromptTitle): The title displayed on the
 # dialog that prompts the user to allow the incoming connection.
 remoteIncomingPromptTitle=Połączenie przychodzące
-# LOCALIZATION NOTE (remoteIncomingPromptMessage): The message displayed on the
-# dialog that prompts the user to allow the incoming connection.
-remoteIncomingPromptMessage=Wykryto prośbę o połączenie umożliwiające zdalne debugowanie. Całkowitą kontrolę nad twoją przeglądarką może przejąć zdalny klient! Pozwolić na połączenie?
-remoteIncomingPromptDisable=Wyłącz
-remoteIncomingPromptAccept=OK
-remoteIncomingPromptCancel=Anuluj
+# LOCALIZATION NOTE (remoteIncomingPromptUSB): The message displayed on the
+# dialog that prompts the user to allow an incoming USB connection.
+remoteIncomingPromptUSB=Pozwolić na połączenie debugowania przez USB?
+# LOCALIZATION NOTE (remoteIncomingPromptUSB): The message displayed on the
+# dialog that prompts the user to allow an incoming TCP connection.
+remoteIncomingPromptTCP=Pozwolić na zdalne połączenie debugowania z %1$S:%2$S? Wymaga ono zeskanowania kodu QR, aby uwierzytelnić certyfikat zdalnego urządzenia. Można zapamiętać urządzenie, aby uniknąć powtórnego skanowania.
+# LOCALIZATION NOTE (remoteIncomingPromptDeny): This button will deny an
+# an incoming remote debugger connection.
+remoteIncomingPromptDeny=Odmów
+# LOCALIZATION NOTE (remoteIncomingPromptAllow): This button will allow an
+# an incoming remote debugger connection.
+remoteIncomingPromptAllow=Pozwól
+# LOCALIZATION NOTE (remoteIncomingPromptScan): This button will start a QR
+# code scanner to authenticate an incoming remote debugger connection.  The
+# connection will be allowed assuming the scan succeeds.
+remoteIncomingPromptScan=Skanuj
+# LOCALIZATION NOTE (remoteIncomingPromptScanAndRemember): This button will
+# start a QR code scanner to authenticate an incoming remote debugger
+# connection.  The connection will be allowed assuming the scan succeeds, and
+# the other endpoint's certificate will be saved to skip future scans for this
+# client.
+remoteIncomingPromptScanAndRemember=Skanuj i zapamiętaj
```

Comment:
We are trying not to use "zapamiętaj" as "remember" translation, "zachowaj" should be better in most cases and when talking directly about writing to file also "zapisz".